src/Engine/Graphics/Outdoor.cpp

```diff
@@ -2020,27 +2020,32 @@ double OutdoorLocation::GetPolygonMaxZ(RenderVertexSoft *pVertex, unsigned int u
     return result;
 }
 
+// TODO(pskelton): move this - used both indoors and out
 void TeleportToStartingPoint(MapStartPoint point) {
     std::string pName = toString(point);
 
     if (pDecorationList->GetDecorIdByName(pName) != DECORATION_NULL) {
```

Comment:
Side note: we could avoid repeatedly calling `pDecorationList->GetDecorIdByName(pName)` below.